public/js/index.js

```diff
@@ -979,6 +979,23 @@ function initRepository() {
     }
 }
 
+var toggleMigrations = function(){
+    console.log("toggleMigrations")
+    if (!$('#mirror').is(":checked") && $('#auth_username').val().length > 0 && ($('#clone_addr').val().startsWith("https://github.com") || $('#clone_addr').val().startsWith("http://github.com"))) {
```

Comment:
Won't this only be triggered when a user loads the page and the clone url is already in the clone field? IMHO this should listen on input.